jetty-servlets/src/test/java/org/eclipse/jetty/servlets/QoSFilterTest.java

```diff
@@ -255,7 +253,7 @@ protected void doGet(HttpServletRequest request, HttpServletResponse response) t
                         __maxSleepers = __sleepers;
                 }
 
-                Thread.sleep(50);
+                Thread.sleep(500);
```

Comment:
I can't see how this will actually fix the problem, the failure was coming from `rethrowExceptions(futures)` so I think one of the workers is throwing. But maybe it is worth just enabling the test to see if it is still flaky.

Also I think we should remove the warning in each of the tests and always test:
```java
assertEquals(TestServlet.__maxSleepers, maxQos);
```

No one is going to be checking for warnings in tests so it is not useful.